plugins/com.google.cloud.tools.eclipse.swtbot/src/com/google/cloud/tools/eclipse/swtbot/SwtBotTreeUtilities.java

```diff
@@ -244,6 +244,7 @@ public static boolean hasChild(SWTWorkbenchBot bot, SWTBotTree tree, Matcher<Str
         () -> {
           TreeItem[] items = tree.widget.getItems();
           for (TreeItem item : items) {
+            System.out.println("Debug Item: " + item.getText());
```

Comment:
I assume this is temporary so you can see the output in kokoro?

Reply:
Yeah — my local Windows VM is h0s3d :-/

Reply:
Seems like there isn't a main thread on Windows.  But thing about it, we just want to check that there are threads reported.

Reply:
to be removed